doc/v1/docs/api_projects.md

```diff
@@ -0,0 +1,286 @@
+#Project Api Calls
+
+ARGO Messaging Service supports project entities as a basis of organizing and isolating groups of users & resources
+
+## [POST] Manage Projects - Create new project
+This request creates a new project with the given project_name with a POST request
+
+### Request
+```json
+POST "/v1/projects/{project_name}"
+```
+
+### Where
+- Project_name: Name of the project to create
+
+### Post body:
+```
+json
+{
+  "description" : "a simple description"
+}
+```
+
+### Example request
+```
+json
+curl -X POST -H "Content-Type: application/json"
+ -d POSTDATA "https://{URL}/v1/projects/PROJECT_NEW?key=S3CR3T"
+```
+
+### Responses  
+
+If successful, the response contains the newly created project
+
+Success Response
+`200 OK`
+```
+json
+{
+ "name": "projects/EGI/topics/monitoring"
+}
+```
+
+### Errors
+Please refer to section [Errors](api_errors.md) to see all possible Errors
+
+
+## [DELETE] Manage Topics - Delete topic
```

Comment:
Indeed this is a mispasted line. The rest of project documentation will be added with the implementation of the Update,Delete requests